src/IFTPipeline.jl

```diff
@@ -7,22 +7,35 @@ module IFTPipeline
 using ArgParse
 using LoggingExtras
 using IceFloeTracker
-using IceFloeTracker: Folds, RGB, Gray, load, float64, imsharpen
+using IceFloeTracker: Folds, RGB, Gray, load, float64, imsharpen, label_components
```

Comment:
Might have to update this after merging #31?

Reply:
Probably